src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java

```diff
@@ -230,6 +231,17 @@ int run(String[] args) {
         }
         Path outputPath = null;
         try {
+            try (InputStream savedOptions = JlinkTask.class.getModule().getResourceAsStream("jdk/tools/jlink/internal/options")) {
```

Comment:
nit: wrap this long line that helps future side-by-side review.

Reply:
Nit: define a static variable for this resource file name `jdk/tools/jlink/internal/options` that the plugin can use.